string srcPath
string dstPath
----
float64 confidence